.travis.yml

```diff
@@ -20,18 +20,34 @@ addons:
       - liblapack-dev
       - diffutils
       - findutils
+      - libeigen3-dev
+
+sudo: required
+
+install:
+
+  - wget bitbucket.org/eigen/eigen/get/3.3.5.tar.gz; tar -xf 3.3.5.tar.gz
```

Comment:
Oh, sorry but I won't approve tarballs like that. 
I want arpack to be easy to build, this add too much complexity :/

Reply:
100% agree/understand. But seems no other way (as CI boxes are outdated, eigen version is outdated => CD !). Or this PR will have to be dropped : pity if this happens, but, I do not see another way (merging dead code is not a solution neither...)

Reply:
... Or maybe try to add macos boxes with travis CI. As far as I remember they are a lot more "updated" : it could be possible to turn off arpackmm on outdated ubuntu, but, to turn it on on macos.

I succeeded to do a macos box for another project with travis CI. I am running debian that I know quite well. Not keen on macos. But this is probably something that could be set to work (and would give you twice validations). 

So maybe if there is a macos expert around that would set up the .travvis.yml for macos... At least for a try (?). 

I understand 100% : I know how maintaining is hard work ! :D

Reply:
check out 06a6993 : does this sounds better to you ? (tarball : wget only the very first time, saved/retrieved from the cache)

Reply:
OK, df60777 works (eigen // CI): probably the best I could propose on ubuntu...